libraries/core-react/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -38,13 +36,20 @@ const Collapsed = styled(Typography)`
   text-decoration: none;
 `
 
-export const Breadcrumbs = forwardRef(function Breadcrumbs(
-  { className, children, collapse, ...rest },
+type Props = {
+  /*
+   * Collapses the list of breadcrumbs so that only the first
+   * and last breadcrumb will be shown, with an ellipsis in between.
+   */
+  collapse?: boolean
+} & JSX.IntrinsicElements['nav']
```

Comment:
This is also something we do differently from component to component… `React.HTMLAttributes<HTMLNavElement> `vs. `JSX.IntrinsicElements['nav']`

Reply:
@mimarz and I discussed this yesterday, if I remember correctly the difference was that `JSX.IntrinsicElements['nav']` included more props?

Reply:
@vnys Funfact, [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) only uses the `HTMLElement` dom interface

Reply:
> @vnys Funfact, [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) only uses the `HTMLElement` dom interface

Noted 😉